packages/web-components/src/components/ic-card/ic-card.tsx

```diff
@@ -289,20 +289,27 @@ export class Card {
               <slot name="interaction-controls"></slot>
             </div>
             {expandable && (
-              <ic-button
-                class={{
-                  ["toggle-button"]: true,
-                  ["toggle-button-closed"]: !this.areaExpanded,
-                  ["toggle-button-expanded"]: this.areaExpanded,
-                }}
-                variant="icon"
-                size="large"
-                aria-label="Toggle expandable area"
-                aria-expanded={this.areaExpanded}
-                aria-controls="expanded-content-area"
-                onClick={this.toggleExpanded}
-                innerHTML={chevronIcon}
-              ></ic-button>
+              <ic-tooltip
+                id="ic-tooltip-expand-button"
+                label="Toggle expandable area"
+                silent
+              >
+                <button
+                  class={{
+                    ["toggle-button"]: true,
+                    [`toggle-button-${
+                      this.areaExpanded ? "expanded" : "closed"
+                    }`]: true,
+                  }}
+                  aria-label="Toggle expandable area"
+                  aria-expanded={this.areaExpanded}
```

Comment:
I think you need to set this to explictly "true" or "false" based on this.areaExpanded. Currently when I use Voiceover, it doesn't give me any suggestion that the button expands:
![Screenshot 2024-03-07 at 14 42 08](https://github.com/mi6/ic-ui-kit/assets/111292477/875328f4-7f55-489e-87ee-105b5fe57e4a)